packages/@o3r/core/schematics/ng-add/project-setup/index.ts

```diff
@@ -64,15 +61,11 @@ export const prepareProject = (options: NgAddSchematicsSchema) => async (tree: T
   ]));
   const projectPackageJson = workspaceProject ? readPackageJson(tree, workspaceProject) : tree.readJson('package.json') as PackageJson;
   const externalPackagesToInstallWithNgAdd: Record<string, string | undefined> = {};
-  // Ngx-prefetch version is aligned with angular
-  if (projectType === 'application' && context.interactive) {
-
-    const installPrefetch = await askConfirmation('Activate prefetch builder to generate prefetcher JavaScript file (see https://github.com/AmadeusITGroup/ngx-prefetch)?', false);
-    if (installPrefetch) {
-      const angularVersion = projectPackageJson.dependencies?.['@angular/core'] || projectPackageJson.peerDependencies?.['@angular/core'];
-      const angularMajorVersion = angularVersion?.match(simplifiedSemVerRegexp)?.[1];
-      const ngxPrefetchVersion = angularMajorVersion ? `^${angularMajorVersion}.0.0` : undefined;
-      externalPackagesToInstallWithNgAdd['@o3r/ngx-prefetch'] = ngxPrefetchVersion;
+  if (projectType === 'application') {
+    const ngxPrefetch = '@o3r/ngx-prefetch';
+    const ngxPrefetchVersion = projectPackageJson.dependencies?.[ngxPrefetch];
+    if (typeof ngxPrefetchVersion !== 'undefined') {
+      externalPackagesToInstallWithNgAdd[ngxPrefetch] = ngxPrefetchVersion;
```

Comment:
do we want to keep this at all ? or should it comes from the preset used to create the app ?